IPython/core/debugger.py

```diff
@@ -621,3 +621,13 @@ def do_where(self, arg):
                 self.print_stack_trace()
 
         do_w = do_where
+
+
+def set_trace(frame=None):
```

Comment:
The other `set_trace()` functions also don't have a `frame` parameter, though the `set_trace()` *method* does. I'd either remove it from here, or also add it to the version in `IPython.terminal`. @Carreau, opinions?

Also, if we decide to keep the parameter, it should probably actually do something ;-)